modules/preparing-vsphere-for-windows-containers.adoc

```diff
@@ -0,0 +1,157 @@
+      </component>
+      <component xmlns:wcm="http://schemas.microsoft.com/WMIConfig/2002/State" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" name="Microsoft-Windows-Shell-Setup" processorArchitecture="amd64" publicKeyToken="31bf3856ad364e35" language="neutral" versionScope="nonSxS">
+         <ComputerName>windows-host</ComputerName>
+         <ProductKey>My_Product_key</ProductKey>
+      </component>
+   </settings>
+   <settings pass="oobeSystem">
+      <component xmlns:wcm="http://schemas.microsoft.com/WMIConfig/2002/State" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" name="Microsoft-Windows-Shell-Setup" processorArchitecture="amd64" publicKeyToken="31bf3856ad364e35" language="neutral" versionScope="nonSxS">
+         <AutoLogon>
+            <Password>
+               <Value>MyPassword</Value>
+               <PlainText>true</PlainText>
+            </Password>
+            <Enabled>true</Enabled>
+            <Username>Administrator</Username>
+         </AutoLogon>
+         <OOBE>
+            <HideEULAPage>true</HideEULAPage>
+            <HideLocalAccountScreen>true</HideLocalAccountScreen>
+            <HideOEMRegistrationScreen>true</HideOEMRegistrationScreen>
+            <HideOnlineAccountScreens>true</HideOnlineAccountScreens>
+            <HideWirelessSetupInOOBE>true</HideWirelessSetupInOOBE>
+            <NetworkLocation>Work</NetworkLocation>
+            <ProtectYourPC>1</ProtectYourPC>
+            <SkipMachineOOBE>true</SkipMachineOOBE>
+            <SkipUserOOBE>true</SkipUserOOBE>
+         </OOBE>
+         <RegisteredOrganization>Organization</RegisteredOrganization>
+         <RegisteredOwner>Owner</RegisteredOwner>
+         <DisableAutoDaylightTimeSet>false</DisableAutoDaylightTimeSet>
+         <TimeZone>Eastern Standard Time</TimeZone>
+         <UserAccounts>
+            <AdministratorPassword>
+               <Value>MyPassword</Value>
+               <PlainText>true</PlainText>
+            </AdministratorPassword>
+            <LocalAccounts>
+               <LocalAccount wcm:action="add">
+                  <Description>Administrator</Description>
+                  <DisplayName>Administrator</DisplayName>
+                  <Group>Administrators</Group>
+                  <Name>Administrator</Name>
+               </LocalAccount>
+            </LocalAccounts>
+         </UserAccounts>
+      </component>
+   </settings>
+</unattend>
+----
+====
+
+== vSphere environment changes for Windows container workloads
+
+The WMCO downloads the Ignition files from the internal API server endpoint. To enable this, add a new DNS entry for `api-int.<cluster_name>.<base_domain>` which points to the external API server URL `api.<cluster_name>.<base_domain>`. The external API endpoint was already created as part of the initial cluster installation on vSphere.
+
+[NOTE]
+====
+For Linux nodes, CoreDNS is running on every node which helps in resolving the internal API server URL.
+====
```

Comment:
Is this important info for customers?

Reply:
Since it pertains to Linux nodes, probably not. I'll remove if there's no relevant correlation to Windows nodes.